README.md

```diff
@@ -2,4 +2,159 @@
+
+5. Write the `drawCard()` function to return the next card. It should remove that method from the remaining cards and add it to the dealt cards.
+
+6. Write the `reshuffle()` method to randomize the cards that have not yet been dealt out.  
+**Hint:** *The* `arc4random_uniform()` *C function is accessible in Swift also.*
+
+7. Write the `shuffle()` method that shuffles the dealt cards back into the cards that are ready to deal, randomizing them again.
+
+### `Player.swift`
+
+1. Re-enable the target membership for `PlayerSpec.swift` for `swift-blackjackTests`.
+
+2. Create a new class file for the `Player` class. Have it conform to the `CustomDebugStringConvertible` protocol. It will need nine properties total:
+  * `name`, a string,
+  * `cards`, an array of `Card` objects,
+  * `handscore`, an unsigned integer,
+  * `blackjack`, a boolean,
+  * `busted`, a boolean,
+  * `stayed`, a boolean,
+  * `mayHit`, a boolean,
+  * `wallet`, an unsigned integer, and
+  * `debugDescription`, a string.
+
+3. Write an initializer which takes an argument for the `name` property. Make the initial value of the `wallet` property `100`.
+
+4. Make `debugDescription` a calculated property which returns a string detailing the object's current state.
+
+5. Make `handscore` a calculated property that evaluates the `cards` array. It may use one Ace as an "eleven" (11) if doing so will not bust the hand.
+
+6. Make `blackjack` a calculated property that determines whether the hand is a blackjack (a score of 21 with only two cards).
+
+7. Make `busted` a calculated property that determines whether the hand is busted (over score of 21).
+
+8. Leave the `stayed` property initialized to `false`. It will need to be used to hold state during a round.
+
+9. Make `mayHit` a calculated property that determines whether the player may take a new card (if the hand is not busted, is not a blackjack, and if the player has not stayed).
+
+10. Write a method called `canPlaceBet()` which takes an unsigned integer argument and returns a boolean of whether or not the player can afford the submitted bet.
+
+11. Write two methods called `didWin()` and `didLose()` which both take an unsigned integer for the value of the bet. The `didWin()` method should add the value of the bet argument to the player's wallet, while the `didLose()` method should subtract it.
+
+12. Write a method called `didPush()` which does nothing. Leave a comment within the implementation that this inaction is intentional.
+
+### `House.swift`
+
+1. Re-enable the target membership for `HouseSpec.swift` for `swift-blackjackTests`.
+
+2. Create a new class file for the `House` class. It should be a subclass of the `Player` class.
+
+3. Override the superclass's initializer to set the `wallet` property to `1000`.
+
+4. Add a calculated property `mustHit` that returns a boolean of whether the house must take a new card or not. Treat the "house rules" as staying at a score of seventeen (17).
```

Comment:
it was originally, but i made it a calculated property just because it's a boolean.